lib/browser/parse-features-string.ts

```diff
@@ -26,7 +26,10 @@ const keysOfTypeNumberCompileTimeCheck: { [K in IntegerBrowserWindowOptionKeys]
 };
 // Note `top` / `left` are special cases from the browser which we later convert
 // to y / x.
-const keysOfTypeNumber = new Set(['top', 'left', ...Object.keys(keysOfTypeNumberCompileTimeCheck)]);
+// NOTE(@mlaurencin)`innerWidth` / `innerHeight` are also special cases. They should be equivalent to
```

Comment:
This comment is confusing - which "they" is being used here and what's the expected behavior? Something like:

```
`innerWidth` / `innerHeight` are also special cases. The spec states that `width` and `height` represent the window content size and are equivalent to `innerWidth` / `innerHeight`, but our implementation currently incorrectly maps `width` and `height`  to `outerWidth` and `outerHeight`, or the size of the window with all border and related window chrome.
```

is probably clearer.